JavaScript/JavaScriptSDK.Tests/CheckinTests/Context/Session.tests.ts

```diff
@@ -10,11 +10,19 @@ class SessionContextTests extends TestClass {
     /** Method called before the start of each test method */
     public testInitialize() {
         this.results = [];
+
+        if (window.localStorage) {
```

Comment:
(Aesthetics)
Move these three lines to a separate func so that you can reuse below?